### PR TITLE
Firebasedeploy

### PR DESCRIPTION
**Pull request by DanPGill**

### CODE REVIEW

**Comment by richardgill** [comment]
This is roughly right. You've added an index.html into public. The goal is to make firebase have your React site (which is built into a folder by `yarn build`).

**Comment by richardgill** [comment]
Could you add me to the firebase project (richard@rgill.co.uk)

What's the URL to the page when it's hosted on firebase?

**Review by DanPGill — COMMENTED** [review]
Updated firebase so site deploys at https://reddit-clone-178ed.firebaseapp.com/; removed unnecessary code (todos app, importexport & reacttasks)

File: .firebaserc [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "projects": {
-    "default": "<PROJECT>"
+    "default": "reddit-clone-178ed",
```
Comment (richardgill) [inline .firebaserc:3]: do you need both of these?

**Pull request merged**